sdk/python/foundation-models/system/inference/text-to-image/scoring-files/score/score_batch.py

```diff
@@ -0,0 +1,584 @@
+# Copyright (c) Microsoft. All rights reserved.
```

Comment:
For the missing doc strings - I am assuming, that how it was at the source.

Reply:
Regarding missing doc string and different format, this is coming from the source (i..e, original file from batch team wrapper.) Can we keep it as it is for original code. I have added docstring wherever I have touched in the code.